omnibus/config/software/datadog-agent.rb

```diff
@@ -273,6 +273,10 @@
         mode: 0644,
         vars: { install_dir: install_dir }
 
+    erb source: "gui.launchd.plist.erb",
```

Comment:
Need probably not but I figure it isn't costing much to template it as well and if (for some reason) we need to add substitution it will be already setup.

Reply:
Fair enough it doesn't cost much and it's a build time cost anyway